plugins/discourse-local-dates/assets/javascripts/discourse/templates/components/discourse-local-dates-create-form.hbs

```diff
@@ -156,7 +156,7 @@
       @label="discourse_local_dates.create.form.insert" />
   {{/if}}
 
-  <a class="cancel-action" href {{action "cancel"}}>
+  <a class="cancel-action" href {{action "cancel"}} role="button">
```

Comment:
It's a slightly larger change but could we use `<DButton>` instead?

Reply:
Sure, we don't quite have a style for a flat text button, so I added that as well to the styleguide. Will update the OP with a screenshot.